### DISCUSSION TITLE
Default Gamport 1 Input in Settings

### DISCUSSION THREAD

**Discussion by aZtOcKdOg** [vAmigaWeb/vAmigaWeb#176] [Ideas]
A default checkbox in the Settings GUI (somewhere maybe just above 'Game Controller') for Gameport 1.

(is that still referenced as Amiga HW 'Joy0' in this codebase?) 

![image](https://github.com/vAmigaWeb/vAmigaWeb/assets/64781882/309afa34-4bcf-49c5-92eb-afb023d1f7c5)

'Default Mouse Input'

This can be left 'blank' by default and the user that wishes to set a default (or maybe the last chosen input is remembered like the KS file is)

The other expanded idea on this would be Gameport 1 is empty by default (first load ever) but what is selected in the 'Default Mouse Input' with an option for the defaults could be distinguished by 'active on touch' or 'active on launch'

Example workflow:

-Start vAmigaWeb PWA from iPadOS homescreen
-Insert ADF or HDF
-Touch the emulator screen to start using input method selected in 'Default Mouse Input' (handy for HW mouse, touch mouse etc)

ofc this can be useful for either gameport and ANY input method!

**Comment by mithrendal** [comment]
Or we simple persist the chosen values in both ports on selection ? i.e. no need for default settings in the settings dialog ? On cold start these values are then restored in both ports ?
